### ISSUE TITLE
Verification of usage of roboflow to run cogVLM

### ISSUE THREAD

**Issue by PhilipAmadasun** [THUDM/CogVLM#406]
Is this [link](https://blog.roboflow.com/how-to-deploy-cogvlm/) a legitimate source on how to use cogVLM via roboflow?

**Comment by zRzRzRzRzRzRzR** [comment]
no, cogvlm used and trained in SAT formated and not support finetune through hf and peft. So It will cause some problem using roboflow.

**Comment by PhilipAmadasun** [comment]
@zRzRzRzRzRzRzR Thanks for the reply Is there some way to run cogVLM on an A100 and call it through some Restful API or something? I'm not interested in using the webUI.

**Comment by zRzRzRzRzRzRzR** [comment]
try use openai demo in this repos

**Comment by PhilipAmadasun** [comment]
We will look into this next week, again thank you. Please keep this issue open for now.

**Comment by PhilipAmadasun** [comment]
@zRzRzRzRzRzRzR We looked at your openai demo. Just to make sure we understand what is going on, you aren't actually using the openai API, you just set up a code that kinda works like the openai API right? We have been trying to actually use the open API to run the cogVLM we pull from huggingface, would that work or are we going in the wrong direction?
